Dockerfiles/midstream/Dockerfile

```diff
@@ -8,8 +8,9 @@ RUN export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm6
     curl -L -o /usr/local/bin/operator-sdk ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH} \ 
     && chmod a+x /usr/local/bin/operator-sdk && \
     mkdir /project/output && \
-    dnf install --setopt=install_weak_deps=False -y git-core ansible && \
-    dnf clean all && \
-    git clone --branch $OPERATOR_TEST_PLAYBOOKS_TAG https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
-COPY ./entrypoint.sh /project/entrypoint.sh
-ENTRYPOINT ["/project/entrypoint.sh"]
+    dnf install --setopt=install_weak_deps=False -y git-core ansible skopeo python3-pip dnf-plugins-core && \
```

Comment:
Why do we need to install pip ? We do not install anything with pip in the Dockerfile.
And Dockerfile should ideally contain all necessary dependencies for running tests.
it will reduce random network issues in the execution of tests.

Reply:
Removed installation of pip